kubernetes/configMaps/filterlist.txt

```diff
@@ -0,0 +1,24 @@
+resource://
```

Comment:
I'm not sure how I feel about this duplication. Since we merged in your last PR, there is already an example file that would be nice to somehow reuse.

Reply:
I'm happy enough to remove it either from the top level or from here. If I removed it from here I'd need to provide more instructions in the `README.md`
> copy the sample.filterlist.txt file from root of this repo into the configMaps then add to the custom values file.
Then remove the simple example in kubernetes/values.yaml. Directly re-using the root level sample.filterlist.txt would be a bad idea as the config map is written to take any file in the target directory and create a config map from it. This makes the chart more extensible by allowing other config files to be created for the kubernetes pod(s) by simply putting them into the same directory. 

To make practical use of this collector you either need to extend the functionality in code, or add a second container to read the json output. You're then in a position of 
> What config options does that second container need?

Reply:
Definitely a discussion for another PR but I'll give a quick background here to the existing functionality since it's something that is somewhat critical to this. The `blockedURI` list is intended as a mechanism to drop any reports that are completely unactionable and provide no use to collection. There are a multitude of reasons for these hitting the collector (browser silliness, browser extensions, malicious actors) and they are something outside of the control of the operator so this list ensures that they don't need to worry about it. In the first year of our CSP, we received along the lines of 70% unactionable violation reports which took up storage and compute resources.

> This comes down to, If someone opens the developer console of their browser should they see an accepted report for these or a error?

The console won't show either. The CSP violation reports are in a similar fashion to UDP where the report is fire and forget without any reporting on whether or the payload was accepted.

> This is a known issue that sadly cannot be auctioned. As it tries to load about 3 or 4 fonts per tile it would swamp the reports and so are filtered out before being passed to the visualisation parts of the platform.

I think your approach of loading in a file with the blocked URIs is a great use case for this. We also do sampling on the application itself (by only defining the `report-uri` directive for a percentage of requests) due to the sheer size of our application. Usually we launch a new directive change with a 0.001% as to not flood the log aggregator if something was missed. We later tune it up to a larger portion of the traffic.